src/NuGet.Protocol.Catalog/NuGet.Protocol.Catalog.csproj

```diff
@@ -0,0 +1,98 @@
+    <PackageProjectUrl>https://github.com/NuGet/NuGet.Services.Metadata</PackageProjectUrl>
+    <RepositoryType>git</RepositoryType>
+    <RepositoryUrl>https://github.com/NuGet/NuGet.Services.Metadata</RepositoryUrl>
+  </PropertyGroup>
+  <ItemGroup>
+    <Reference Include="Microsoft.CSharp" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions">
+      <Version>1.0.0</Version>
+    </PackageReference>
+    <PackageReference Include="Newtonsoft.Json">
+      <Version>9.0.1</Version>
+    </PackageReference>
+    <PackageReference Include="NuGet.Build.Tasks.Pack">
+      <Version>4.8.0</Version>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
+      <PrivateAssets>all</PrivateAssets>
+    </PackageReference>
+    <PackageReference Include="NuGet.Protocol">
+      <Version>4.8.0</Version>
+    </PackageReference>
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="CatalogClient.cs" />
+    <Compile Include="CatalogProcessor.cs" />
+    <Compile Include="CatalogProcessorSettings.cs" />
+    <Compile Include="FileCursor.cs" />
+    <Compile Include="ICatalogClient.cs" />
+    <Compile Include="ICatalogLeafProcessor.cs" />
+    <Compile Include="ICursor.cs" />
+    <Compile Include="Models\CatalogIndex.cs" />
+    <Compile Include="Models\CatalogLeaf.cs" />
+    <Compile Include="Models\CatalogLeafItem.cs" />
+    <Compile Include="Models\CatalogLeafType.cs" />
+    <Compile Include="Models\CatalogPage.cs" />
+    <Compile Include="Models\CatalogPageItem.cs" />
+    <Compile Include="Models\ICatalogLeafItem.cs" />
+    <Compile Include="Models\ModelExtensions.cs" />
+    <Compile Include="Models\PackageDeleteCatalogLeaf.cs" />
+    <Compile Include="Models\PackageDependency.cs" />
+    <Compile Include="Models\PackageDependencyGroup.cs" />
+    <Compile Include="Models\PackageDetailsCatalogLeaf.cs" />
+    <Compile Include="Properties\AssemblyInfo.cs" />
+    <Compile Include="Properties\AssemblyInfo.*.cs" />
+    <Compile Include="Serialization\BaseCatalogLeafConverter.cs" />
+    <Compile Include="Serialization\CatalogJsonSerialization.cs" />
+    <Compile Include="Serialization\CatalogLeafItemTypeConverter.cs" />
+    <Compile Include="Serialization\CatalogLeafTypeConverter.cs" />
+  </ItemGroup>
+  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+  <PropertyGroup>
+    <SignPath>..\..\build</SignPath>
+    <SignPath Condition="'$(BUILD_SOURCESDIRECTORY)' != ''">$(BUILD_SOURCESDIRECTORY)\build</SignPath>
+    <SignPath Condition="'$(NuGetBuildPath)' != ''">$(NuGetBuildPath)</SignPath>
+    <SignType Condition="'$(SignType)' == ''">none</SignType>
+  </PropertyGroup>
+  <Import Project="$(SignPath)\sign.targets" Condition="Exists('$(SignPath)\sign.targets')" />
+  <Import Project="$(SignPath)\sign.microbuild.targets" Condition="Exists('$(SignPath)\sign.microbuild.targets')" />
+  <Import Project="..\..\sign.thirdparty.targets" />
```

Comment:
I would double check that all of the dependencies of this project are in the third-party list. May want to kick off a release build on this feature branch just to make sure that `Extract and Verify Artifacts` succeeds.

Reply:
Good idea.

Reply:
I forgot MicroBuild. Should be good now.